front/lib/dfs/index.ts

```diff
@@ -0,0 +1,52 @@
+import type { Bucket } from "@google-cloud/storage";
```

Comment:
`gcs` rather than `dfs` in name no?

Reply:
I stick with DFS for this file/class but all other occurrences will go with GCS.